templates/releaseTemplate.yml

```diff
@@ -54,11 +54,16 @@ stages:
 
       - checkout: self
 
-      - checkout: _LansaAWSTemplates
-        displayName: Checkout aws-templates-md
+      - checkout: _lansa_aws-templates
+        displayName: Checkout _lansa_aws-templates
+        path: "s/_lansa_aws-templates"
 
-      - checkout: _priyadishahCookbooks
-        displayName: Checkout cookbooks
+      # - checkout: _priyadishahCookbooks
+      #   displayName: Checkout cookbooks
+
+      - checkout: _robe070_cookbooks
+        displayName: Rob cookbooks
+        path: "s/_robe070_cookbooks"
```

Comment:
Curious path "s/"?

Reply:
it was not picking correct path, worked with tarun he asked me add this

Reply:
removed "S/" from path and it is resolved.